root/static/js/userpages/UserPage.js

```diff
@@ -25,6 +26,22 @@ app.controller('UserPageController', function($scope, $http, fn) {
 	$scope.count = {};
 	$scope.topics = [];
 
+    $scope.imgUrl = 'https://duckduckgo.com/t/userpage_' + $scope.response.gh_data.id;
+    $scope.randomNum = Math.ceil(Math.random() * 1e7);
+    $scope.imgUrlSuffix = '?' + Math.ceil(Math.random() * 1e7);
+
+    $scope.newImg = new Image();
+    $scope.newImg.src = $scope.imgUrl + $scope.imgUrlSuffix;
```

Comment:
Could you just replace this with a call to `$scope.addImg($scope.response.gh_data.id)`?